npc/re/merchants/enchan_verus.txt

```diff
@@ -1094,6 +1094,7 @@ verus04,163,219,4	script	PLUTO_09#pa0829	4_SCR_AT_ROBOTS,{
 				.@string$ = "Production design (^ff0000 success rate 30%^000000)";
 			mes "[PLUTO_09]";
 			mes "If you give me 5 " + getitemname(6962) + ", I can do the production of design drawings look!";
+			mes "If it fails, 5 " + getitemname(6962) + " will be refunded.";
```

Comment:
I think the message can be simplified to "If it fails, they will be refunded." as the item and quantity is above the current message rather than on a separate page.